providers/summary/scancode.js

```diff
@@ -90,10 +90,22 @@ class ScanCodeSummarizer {
     }
   }
 
+  summarizeDeclaredLisense(files, declaredLicenses) {
+    files.forEach(file => {
+      const asserted = get(file, 'packages[0].asserted_licenses')
+      if (asserted) this._addArrayToSet(asserted, declaredLicenses, license => license.license || license.spdx_license_key)
+      this._addLicenseFiles(file, declaredLicenses)
+    })
+  }
+
   _addLicenseFiles(file, declaredLicenses) {
     // Look for license files at the root of the scanned code
     // TODO enhance in the future to cover more license management strategies.
-    if (!['license', 'license.txt', 'license.md', 'license.html'].includes(file.path.toLowerCase())) return
+    const license = ['license', 'license.txt', 'license.md', 'license.html'].reduce((current, prefix) => {
```

Comment:
suggest calling this `isLicense`. `is`, `has` prefix is a good convention for boolean variables, props and functoins